joomla/joomla-cms
# [4.1] Finder: Fixing Content Map field

### Summary of Changes
This is about the "Content Map" filter field in the "Index" view of Smart Search. While changing the taxonomy table to a nested set schema, I forgot to convert this form field. This means that right now it returns more or less garbage. It should display for example the whole category tree properly indented in the dropdown.

### Testing Instructions
1. Install Joomla with some data, for example the testing sampledata.
2. Go to "Smart Search -> Index"
3. Open the filter dropdown and the "Content Map" filter. The entries are rather jumbled.
4. Apply this PR.
5. Reload the page and check the filter again. Now everything is aligned properly and indented properly.
